DOC: fill in missing docstrings for Series.all and Series.sort

natmokval opened the pull request:
xref #71

MarcoGorelli commented:
Thanks! Though `_pandas_like` is private and doesn't show up in the docs, I'd suggest we instead focus on making sure that everything public has a docstring (so, the files directly under the narwhals folder)

natmokval commented:
> Thanks! Though _pandas_like is private and doesn't show up in the docs, I'd suggest we instead focus on making sure that everything public has a docstring (so, the files directly under the narwhals folder)
>

thank you, it's my mistake. I moved docstrings from private `_pandas_like` to public `series` and added examples.

MarcoGorelli reviewed (approved):
thanks @natmokval !

![](https://i.fluffy.cc/VKWgLLbr8ZGc8Gs0b6Hm8wb5xQGmlbwR.gif)
